lang/python/python-dateutil/Makefile

```diff
@@ -15,33 +15,51 @@ PKG_LICENSE:=BSD-2-Clause
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/python-dateutil
 PKG_HASH:=88f9287c0174266bb0d8cedd395cfba9c58e87e5ad86b2ce58859bc11be3cf02
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-dateutil-$(PKG_VERSION)
 
+include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
+
 include ../python-package.mk
+include ../python3-package.mk
 
-define Package/python-dateutil
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+HOST_UNPACK:=$(HOST_TAR) -C $(HOST_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
```

Comment:
same comment for `HOST_UNPACK` as https://github.com/openwrt/packages/pull/7900/commits/c38f4b20e6691a838f4b84b88fefc24c148f0c95#r245578181

Reply:
and ditto.